docs/source/snapshot.rst

```diff
@@ -106,39 +106,44 @@ Requirement Types (``requirement_types``)
     system_requirement:
       long_name: System Requirement
       attributes: # Field Definitions, we don't need the IDs
-        Capella ID: # Field name
+        capellaID: # Field identifier
```

Comment:
It might become interesting to track the snapshot format version. But that's more something to keep in mind for the future, when (if) this project becomes more widely used, _and_ if there are other non-backwards-compatible changes in the format.